lib/useful/append.js

```diff
@@ -0,0 +1,28 @@
+import Promise from 'bluebird';
+import sentry from '../config/sentry';
+
+function append(bot, msg, suffix) {
+  if (!suffix) {
+    bot.sendMessage(msg.channel, 'Usage: **`!!`** `text`');
+    return;
+  }
+
+  Promise.resolve(msg.channel.messages.getAll('author', msg.author))
+    .then(messages => {
+      messages.pop();
+      if (messages[0] === undefined) {
+        return;
+      }
+
+      let newCmd = messages.pop().content.trim().concat(suffix);
+      bot.sendMessage(msg.channel, newCmd);
```

Comment:
That was my original strategy for this command, and it worked when I ran the bot. However having the line `import commands from '../../';` immediately causes `npm run mocha` to give a TypeError. I spent all of last night reading over the error message trying to figure out why and couldn't figure it out. [http://pastebin.com/HY8fpy1q](url) Any idea what's going on?